src/components/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -82,7 +103,19 @@ export const TableHeaderCell = ({
       scope={scope}
       {...other}
     >
-      {children}
+      <div className={clsx(styles['table__header-cell--contents'])}>
+        {children}
+        {sortable && (
```

Comment:
Let's wrap button around this whole table cell content. That way they have a bigger click target, etc.

```
{sortable ? (
<button>
 {children}
 <Icon>
 </button>
 ) : ( 
{children}
)
}
)}
```